### PR TITLE
Micah jones

### PR DESCRIPTION
**Pull request by micahjones13**
Finished MVP, finished stretch animations, added sticky nav

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <!doctype html>
```
Comment (maks112v) [inline index.html:1]: Everything looks good

File: less/mixins.less [hunk 2]
```diff
@@ -1 +1,43 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.center(@justify, @align) {
```
Comment (maks112v) [inline less/mixins.less:2]: maybe add default to not have to always pass in

File: less/mixins.less [hunk 3]
```diff
@@ -1 +1,43 @@
...
+    display: flex;
+    justify-content: @justify;
+    align-items: @align;
+}
+
+.direction(@flex-direction) {
+    flex-direction: @flex-direction;
+}
+
+.btn-style(@width, @border, @background, @border-radius, @height, @color, @font-size, @text-align) {
```
Comment (maks112v) [inline less/mixins.less:12]: You don't need to pass in all of them. Also use `@background: #fffff` to set defaults

File: less/mixins.less [hunk 4]
```diff
@@ -1 +1,43 @@
...
+    color: @color;
+    font-size: @font-size;
+    text-align: @text-align;
+}
+
+.width-shift() {
+    width: 100%;
+}
+
+@keyframes color-shift {
```
Comment (maks112v) [inline less/mixins.less:27]: Good job on the animation

**Review by maks112v — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -33,15 +33,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
...
+    Adaptive relates to using server logic to tell what size screen is being used, responisve uses media queires to detect the device being used.
 2. Describe what it means to be mobile first vs desktop first.
-
+    Mobile first means building from the small end of screen size (mobile) and up, also using min-width instead of max width. Desktop is opposite, going larger down, using max-width.
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
-
+    It makes the rem units factors of 10, because the default text size is 16 px. Doing this makes the default 10px.
 4. How would you describe preprocessing to someone new to CSS?
-
+    It's a version of CSS that is smarter and better and has more options. It takes what you give it and translates it to regular CSS.
```
Comment (Joshua-Wood39) [inline README.md:42]: Very close with the 'translates' answer. The preprocessor compiles your code; so, as you're working on it, you can use a more streamline format (readable and easy to apply), and then the CSS is compiled to its machine-readable format. Good work

**Pull request merged**